libraries/AP_ADSB/AP_ADSB_uAvionix_MAVLink.cpp

```diff
@@ -0,0 +1,27 @@
+/*
+   This program is free software: you can redistribute it and/or modify
+   it under the terms of the GNU General Public License as published by
+   the Free Software Foundation, either version 3 of the License, or
+   (at your option) any later version.
+
+   This program is distributed in the hope that it will be useful,
+   but WITHOUT ANY WARRANTY; without even the implied warranty of
+   MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+   GNU General Public License for more details.
+
+   You should have received a copy of the GNU General Public License
+   along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+#include "AP_ADSB_uAvionix_MAVLink.h"
+
+#if HAL_ADSB_ENABLED

```

Comment:
We could go more fine-grained here - so `HAL_ADSB_UAVIONIX_MAVLINK`.  Initialised to `HAL_ADSB_ENABLED` if it isn't otherwise defined.

Reply:
Hmm... Seems a bit excessive but sure